app-i18n/fcitx-bamboo/fcitx-bamboo-1.0.1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg cmake
+
+MY_PN="fcitx5-bamboo"
+MY_P="${MY_PN}-${PV}"
+S="${WORKDIR}/${MY_PN}-${PV}"
+DESCRIPTION="Typing Vietnamese by Bamboo core engine for Fcitx5"
+HOMEPAGE="https://github.com/fcitx/fcitx5-bamboo"
+
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_P}.tar.xz"
```

Comment:
You still forget the `-> ${P}.tar.xz`

Should be

```
SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_P}.tar.xz -> ${P}.tar.xz"
```

to make the distfile more clear.

Reply:
I defined MY_P at #9

Reply:
The MY_P you defined is used to compose the URL, but the filename of the downloaded file is unpredictable. For now is `fcitx5-bamboo-1.0.1.tar.xz` but maybe changed in the feature (depends on upstream maintainer, impossible in most cases but not impossible).

I just want the filename of the distfile stored locally is predictable.

This is just a suggestion, if you insist on not modifying it, it doesn't matter.